gateway-incoming.yaml

```diff
@@ -191,6 +191,10 @@ components:
             independent results.
           items:
             $ref: '#/components/schemas/AssessmentResult'
+        status:
```

Comment:
Next three lines are indented extra bigly compared to others at same YAML depth.